src/server/middleware/twitter.middleware.ts

```diff
@@ -55,12 +63,13 @@ export async function getTweets(
 
   if (fs.existsSync('src/server/twitter_sample_results.json')) {
     twitterDataPromise = loadLocalTwitterData();
-  } else {
-    if (!enterpriseSearchCredentialsAreValid(apiCredentials)) {
-      res.send(new Error('Invalid Twitter Enterprise Search API credentials'));
-      return;
-    }
+  } else if (v2SearchCredentialsAreValid(apiCredentials)) {
```

Comment:
Optional depending on your preference: Should we print out the API we are using here to the console, in case the user has credentials for both and isn't sure about which we're using? I know this is probably an edge case, but I think it would be helpful.